packages/Toast/src/Toast.styles.js

```diff
@@ -0,0 +1,100 @@
+import styled, { css } from "styled-components";
+import tinycolor from "tinycolor2";
+
+import stylers from "@paprika/stylers";
+import tokens from "@paprika/tokens";
+import Button from "@paprika/button";
+import { visuallyHidden } from "@paprika/stylers/lib/includes";
+
+function getStylesForType(type) {
+  if (type === "visually-hidden") return "";
+  const primaryColors = {
+    success: tokens.color.greenLighten50,
+    warning: tokens.color.yellowLighten30,
+    error: tokens.color.orangeLighten40,
+    info: tokens.color.cremeLighten5,
+    locked: tokens.color.yellowLighten30,
+  };
+
+  return css`
+    background-color: ${primaryColors[type]};
+    border-color: ${tinycolor(primaryColors[type])
+      .darken(10)
+      .toString()};
+    color: ${tinycolor(primaryColors[type]).toHsl().l > 0.7 ? tokens.color.black : tokens.color.white};
+  `;
+}
+
+const AlertLineHeight = "1.3";
+
+const stickyStyles = css`
+  left: 50%;
+  max-width: 675px;
```

Comment:
Should this be `min-width`? Should `max-width` be more like `calc(100%-32px)`?

Reply:
Double checked, it's fixed `min-width` in acl-ui. I think the idea was toast could be very long like this:
![image](https://user-images.githubusercontent.com/38733362/63391830-f0793080-c368-11e9-87ff-a811de47120e.png)

Reply:
Still verifying with designers?